modules/quick-start-content-guidelines.adoc

```diff
@@ -0,0 +1,165 @@
+----
+** *Incorrect example*:
++
+----
+This quick start shows you how to deploy a sample application to {product-title}.
+----
+* The introduction should be a maximum of 4 to 5 sentences, depending on how complex the feature is. A long introduction can overwhelm the user.
+* List the quick start tasks after the introduction content, and start each task with a verb. Do not specify the number of tasks because the copy would need to be updated every time a task is added or removed.
+** *Correct example*:
++
+----
+Tasks to complete: Create a serverless application; Connect an event source; Force a new revision
+----
+** *Incorrect example*:
++
+----
+You will complete these 3 tasks: Creating a serverless application; Connecting an event source; Forcing a new revision
+----
+
+[id="quick-start-content-guidelines-task-steps_{context}"]
+== Task steps
+
+Once the user clicks *Start*, a series of steps appears that they must perform to complete the quick start.
+
+Follow these general guidelines when writing task steps, in addition to the linked style guides at the beginning of this section:
+
+* Use "Click" for buttons and labels. Use "Select" for checkboxes, radio buttons, and drop-down menus.
+* Use "Click" instead of "Click on"
+** *Correct example*:
++
+----
+Click OK.
+----
+** *Incorrect example*:
++
+----
+Click on the OK button.
+----
+
+* Tell users how to navigate between *Administrator* and *Developer* perspectives. Even if you think a user might already be in the appropriate perspective, give them instructions on how to get there so that they are definitely where they need to be.
++
+Examples:
++
+----
+Enter the Developer perspective: In the main navigation, click the dropdown menu and select Developer.
+Enter the Administrator perspective: In the main navigation, click the dropdown menu and select Admin.
+----
+
+* Use the "Location, action" structure. Tell a user where to go before telling them what to do.
+** *Correct example*:
++
+----
+In the node.js deployment, hover over the icon.
+----
+** *Incorrect example*:
++
+----
+Hover over the icon in the node.js deployment.
+----
+
+* Keep your product terminology capitalization consistent.
+* If you must specify a menu type or list as a dropdown, write "dropdown” as one word without a hyphen.
+* Clearly distinguish between a user action and additional information on product functionality.
+** *User action*:
++
+----
+Change the time range of the dashboard by clicking the dropdown menu and selecting time range.
+----
+** *Additional information*:
++
+----
+To look at data in a specific time frame, you can change the time range of the dashboard.
+----
+
+* Avoid directional language, like In the top-right corner, click the icon. Directional language becomes outdated every time UI layouts change. Also, a direction for desktop users might not be accurate for users with a different screen size. Instead, identify something using its name.
```

Comment:
s/In the top-right corner, click the icon/"In the top-right corner, click the icon"/